jobs/rocoto/prepoceanobs.sh

```diff
@@ -14,8 +14,10 @@ export jobid="${job}.$$"
 
 ###############################################################
 # setup python path for class defs and utils
-
-PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${HOMEgfs}/ush"
+# shellcheck disable=SC2311
+pyiodaPATH="${HOMEgfs}/sorc/gdas.cd/build/lib/python$(detect_py_ver)/"
```

Comment:
Would it make more sense to link this path into `${HOMEgfs}/ush/pygfs` as part of `link_workflow.sh`?

Reply:
It seems fine to me as is @DavidHuber-NOAA . This is just copy/pasted from the other "prep" obs scripts, see ``` prepatmiodaobs.sh``` and ```prepsnowobs.sh```

Reply:
Alright, I'm good with this solution.